configure

```diff
@@ -3764,16 +3764,29 @@ return f (e, argv, 0) != argv[0]  ||  f (e, argv, 1) != argv[1];
   return 0;
 }
 _ACEOF
-for ac_arg in '' -qlanglvl=extc89 -qlanglvl=ansi -std \
-	-Ae "-Aa -D_HPUX_SOURCE" "-Xc -D__EXTENSIONS__"
-do
-  CC="$ac_save_CC $ac_arg"
-  if ac_fn_c_try_compile "$LINENO"; then :
-  ac_cv_prog_cc_c89=$ac_arg
+if [[ "$CC" =~ 'ibm-clang' ]] && [[ "$CXX" =~ 'ibm-clang' ]]; then
```

Comment:
Is it possible `$CC` to `=~` `ibm-clang` but `$CXX` to not (or vice versa)? As this stands, it will set flags in the `else` branch which may not be valid if `$CC` and `$CXX` don't match.

Reply:
I don't think we're supposed to mix and match C/C++ compilers, but I think given what's going on in the body of the loop, just checking `CC` probably makes more sense.